packages/prisma/selects/event-types.ts

```diff
@@ -84,6 +84,7 @@ export const availiblityPageEventTypeSelect = Prisma.validator<Prisma.EventTypeS
   userId: true,
   slug: true,
   minimumBookingNotice: true,
+  minimumBookingNoticeType: true,
```

Comment:
not quite sure why we need a boolean here

Reply:
This is to tell prisma that we want that value to go to the backend